.github/workflows/pre-merge-checks.yml

```diff
@@ -29,7 +29,11 @@ jobs:
       - name: Use Node.js 20.x
         uses: actions/setup-node@v4
         with:
-          node-version: 20.11.1
+          node-version: 20.11.sef
+      - name: Setup .npmrc
+        run: |
+          cp .npmrc.template .npmrc && \
+          sed -i s/TOKEN_WITH_READ_PACKAGE_PERMISSION/${{ secrets.GITHUB_TOKEN }}/ .npmrc
```

Comment:
I don't have access to the repo secrets so could someone with access add an access token under `GITHUB_TOKEN` for me?

Reply:
I think this should be provided automatically by github... Looks like there's a different weird error going on before it gets here though 🤔

Reply:
you might need the packages permission? 
https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#about-scopes-and-permissions-for-package-registries
Also I'm not sure what .sef is - the action doesn't seem to like it though

Reply:
Already spoke with @thebauSoftwire but for posterity core-front needs adding to the allow list on the vocab repo. Needs someone with the correct perms to do it though.